miso-web/src/main/webapp/scripts/sample_hot.js

```diff
@@ -640,45 +656,87 @@ Sample.hot = {
         source: Sample.hot.sciName,
         validator: requiredText,
         extraneous: true,
-        include: true
+        include: !isDetailed || show['Tissue']
+      },
+
+      // Parent columns
+      {
+        header: 'Parent Alias',
+        data: 'parentAlias',
+        type: 'text',
+        readOnly: true,
+        include: isDetailed && action == 'propagate'
+      },
+      {
+        header: 'Parent Sample Class',
+        data: 'parentSampleClassAlias',
+        type: 'dropdown',
+        trimDropdown: false,
+        source: Sample.hot.getSampleClasses(),
+        readOnly: true,
+        include: isDetailed && action == 'propagate'
+      },
+
+      // Identity columns
+      {
+        header: 'External Name',
+        data: 'externalName',
+        type: 'text',
+        validator: requiredText,
+        include: show['Identity']
+      },
+      {
+        header: 'Sex',
+        data: 'donorSex',
+        type: 'dropdown',
+        trimDropdown: false,
+        source: Sample.hot.getDonorSexes(),
+        validator: permitEmpty,
+        include: show['Identity']
+      },
+
+      // Detailed columns
+      {
+        header: 'Sample Class',
+        data: 'sampleClassAlias',
+        type: 'dropdown',
+        trimDropdown: false,
+        source: sourceSampleCategory == null ? Sample.hot.getNewSampleClassOptionsAliasOnly() : Sample.hot.getSampleClasses(),
+        include: isDetailed
       },
       {
         header: 'Group ID',
         data: 'groupId',
         type: 'numeric',
         validator: validateNumber,
-        include: detailedBool
+        include: isDetailed
       },
       {
         header: 'Group Desc.',
         data: 'groupDescription',
         type: 'text',
         validator: permitEmpty,
-        include: detailedBool
+        include: isDetailed
       },
       {
-        header: 'External Name',
-        data: 'externalName',
-        type: 'text',
-        validator: requiredText,
-        include: detailedBool && idColBoolean
-      },{
-        header: 'Sex',
-        data: 'donorSex',
+        header: 'Kit',
```

Comment:
Can you confirm that it's the Aliquots which have the kits applied to them, and not the Stocks? I would have thought that an extraction kit (chemical change) happens at the stock level, not the aliquot level.